src/components/CheckoutForm/index.tsx

```diff
@@ -111,7 +111,7 @@ const CheckoutForm: FC<CheckoutFormProps> = ({
       window.Frames.removeAllEventHandlers(window.Frames.Events.PAYMENT_METHOD_CHANGED);
       window.Frames.removeAllEventHandlers(window.Frames.Events.FRAME_ACTIVATED);
     };
-  }, []);
+  }, [onCardValidationChanged, onCardSubmitted, onCardTokenized, paymentMethodChanged, onValidationChanged, onCardTokenizationFailed]);
```

Comment:
in case of second fire for effect, you will have double the event listeners. Not cool

Reply:
function in useeffect returns a cleanup callback that removes all listeners